sample.yml

```diff
@@ -1,13 +0,0 @@
-
-functions:
-  peoplesoft:
-    requirements: ./src/peoplesoft_adapter/requirements.txt
```

Comment:
I don't know whether having this stuff in constitutes an information leak.  It kind of gives a hint at what we try to support internally, which anyone can guess, but even so I am not sure about including it.

Reply:
I've updated the sample file to be `sample_manifest.yml`. The example in the new file is more meaningful.